.github/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+
+<!-- Thanks for submitting this pull request for KeY -->
```

Comment:
I would also add a note:

> By making this pull request, your contribution will be licensed under GPL2-or-later.

Reply:
Is this necessary? Contributing to a GPL project seems to imply this automatically, I think.

Reply:
I am not sure. But the a little difference is, that now the author also acknowledges a re-licensing to GPL3.

Reply:
It's in the [GitHub Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#6-contributions-under-repository-license). I am pretty sure that every user on GitHub needed to accept those.

Reply:
If anything, we can add a link to our license. However, a hard-coded (static) reference is not a good idea, in my opinion.

Reply:
My vote would be to omit this here if possible.